### ISSUE TITLE
Bug before create the world

### ISSUE THREAD

**Issue by minifit123** [refinedmods/refinedstorage#3337]
### Describe the bug

When I create the world, starting the Infinite loading of the world at 100%. 

### How can we reproduce this bug or crash?

-IDK, there is crash not been before, and I changed nothing in mod packs

### What Minecraft version is this happening on?

Minecraft 1.16.5

### What Forge version is this happening on?

36.2.34 and 36.2.35

### What Refined Storage version is this happening on?

1.9.18

### Relevant log output

```shell
[22:20:40] [Server thread/INFO] [on.bl.BlockDrops/]: Block drop calculation started...
[22:21:58] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network node present at BlockPos{x=0, y=0, z=0}, consider removing the block at this position) while calculating drops for Block{refinedstorage:1k_storage_block}
[22:21:58] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network node present at BlockPos{x=0, y=0, z=0}, consider removing the block at this position) while calculating drops for Block{refinedstorage:4k_storage_block}
[22:21:58] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network node present at BlockPos{x=0, y=0, z=0}, consider removing the block at this position) while calculating drops for Block{refinedstorage:16k_storage_block}
[22:21:58] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network node present at BlockPos{x=0, y=0, z=0}, consider removing the block at this position) while calculating drops for Block{refinedstorage:64k_storage_block}
[22:21:58] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network node present at BlockPos{x=0, y=0, z=0}, consider removing the block at this position) while calculating drops for Block{refinedstorage:creative_storage_block}
[22:21:58] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network node present at BlockPos{x=0, y=0, z=0}, consider removing the block at this position) while calculating drops for Block{refinedstorage:64k_fluid_storage_block}
[22:21:58] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network node present at BlockPos{x=0, y=0, z=0}, consider removing the block at this position) while calculating drops for Block{refinedstorage:256k_fluid_storage_block}
[22:21:58] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network node present at BlockPos{x=0, y=0, z=0}, consider removing the block at this position) while calculating drops for Block{refinedstorage:1024k_fluid_storage_block}
[22:21:58] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network node present at BlockPos{x=0, y=0, z=0}, consider removing the block at this position) while calculating drops for Block{refinedstorage:4096k_fluid_storage_block}
[22:21:58] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network node present at BlockPos{x=0, y=0, z=0}, consider removing the block at this position) while calculating drops for Block{refinedstorage:creative_fluid_storage_block}
[22:22:03] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network present at BlockPos{x=0, y=0, z=0}) while calculating drops for Block{refinedstorage:white_controller}[energy_type=on]
[22:22:03] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network present at BlockPos{x=0, y=0, z=0}) while calculating drops for Block{refinedstorage:orange_controller}[energy_type=on]
[22:22:03] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network present at BlockPos{x=0, y=0, z=0}) while calculating drops for Block{refinedstorage:magenta_controller}[energy_type=on]
[22:22:03] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network present at BlockPos{x=0, y=0, z=0}) while calculating drops for Block{refinedstorage:controller}[energy_type=on]
[22:22:03] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network present at BlockPos{x=0, y=0, z=0}) while calculating drops for Block{refinedstorage:yellow_controller}[energy_type=on]
[22:22:03] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network present at BlockPos{x=0, y=0, z=0}) while calculating drops for Block{refinedstorage:lime_controller}[energy_type=on]
[22:22:03] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network present at BlockPos{x=0, y=0, z=0}) while calculating drops for Block{refinedstorage:pink_controller}[energy_type=on]
[22:22:03] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network present at BlockPos{x=0, y=0, z=0}) while calculating drops for Block{refinedstorage:gray_controller}[energy_type=on]
[22:22:03] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network present at BlockPos{x=0, y=0, z=0}) while calculating drops for Block{refinedstorage:light_gray_controller}[energy_type=on]
[22:22:03] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network present at BlockPos{x=0, y=0, z=0}) while calculating drops for Block{refinedstorage:cyan_controller}[energy_type=on]
[22:22:03] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network present at BlockPos{x=0, y=0, z=0}) while calculating drops for Block{refinedstorage:purple_controller}[energy_type=on]
[22:22:03] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network present at BlockPos{x=0, y=0, z=0}) while calculating drops for Block{refinedstorage:blue_controller}[energy_type=on]
[22:22:03] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network present at BlockPos{x=0, y=0, z=0}) while calculating drops for Block{refinedstorage:brown_controller}[energy_type=on]
[22:22:03] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network present at BlockPos{x=0, y=0, z=0}) while calculating drops for Block{refinedstorage:green_controller}[energy_type=on]
[22:22:03] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network present at BlockPos{x=0, y=0, z=0}) while calculating drops for Block{refinedstorage:red_controller}[energy_type=on]
[22:22:03] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network present at BlockPos{x=0, y=0, z=0}) while calculating drops for Block{refinedstorage:black_controller}[energy_type=on]
[22:22:12] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network node present at BlockPos{x=0, y=0, z=0}, consider removing the block at this position) while calculating drops for Block{refinedstorage:white_crafter}[connected=false,direction=east]
[22:22:12] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network node present at BlockPos{x=0, y=0, z=0}, consider removing the block at this position) while calculating drops for Block{refinedstorage:orange_crafter}[connected=false,direction=east]
[22:22:12] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network node present at BlockPos{x=0, y=0, z=0}, consider removing the block at this position) while calculating drops for Block{refinedstorage:magenta_crafter}[connected=false,direction=east]
[22:22:12] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network node present at BlockPos{x=0, y=0, z=0}, consider removing the block at this position) while calculating drops for Block{refinedstorage:crafter}[connected=false,direction=east]
[22:22:12] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network node present at BlockPos{x=0, y=0, z=0}, consider removing the block at this position) while calculating drops for Block{refinedstorage:yellow_crafter}[connected=false,direction=east]
[22:22:12] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network node present at BlockPos{x=0, y=0, z=0}, consider removing the block at this position) while calculating drops for Block{refinedstorage:lime_crafter}[connected=false,direction=east]
[22:22:12] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network node present at BlockPos{x=0, y=0, z=0}, consider removing the block at this position) while calculating drops for Block{refinedstorage:pink_crafter}[connected=false,direction=east]
[22:22:12] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network node present at BlockPos{x=0, y=0, z=0}, consider removing the block at this position) while calculating drops for Block{refinedstorage:gray_crafter}[connected=false,direction=east]
[22:22:12] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network node present at BlockPos{x=0, y=0, z=0}, consider removing the block at this position) while calculating drops for Block{refinedstorage:light_gray_crafter}[connected=false,direction=east]
[22:22:12] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network node present at BlockPos{x=0, y=0, z=0}, consider removing the block at this position) while calculating drops for Block{refinedstorage:cyan_crafter}[connected=false,direction=east]
[22:22:12] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network node present at BlockPos{x=0, y=0, z=0}, consider removing the block at this position) while calculating drops for Block{refinedstorage:purple_crafter}[connected=false,direction=east]
[22:22:12] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network node present at BlockPos{x=0, y=0, z=0}, consider removing the block at this position) while calculating drops for Block{refinedstorage:blue_crafter}[connected=false,direction=east]
[22:22:12] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network node present at BlockPos{x=0, y=0, z=0}, consider removing the block at this position) while calculating drops for Block{refinedstorage:brown_crafter}[connected=false,direction=east]
[22:22:12] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network node present at BlockPos{x=0, y=0, z=0}, consider removing the block at this position) while calculating drops for Block{refinedstorage:green_crafter}[connected=false,direction=east]
[22:22:12] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network node present at BlockPos{x=0, y=0, z=0}, consider removing the block at this position) while calculating drops for Block{refinedstorage:red_crafter}[connected=false,direction=east]
[22:22:12] [Server thread/INFO] [on.bl.BlockDrops/]: Error (IllegalStateException : No network node present at BlockPos{x=0, y=0, z=0}, consider removing the block at this position) while calculating drops for Block{refinedstorage:black_crafter}[connected=false,direction=east]
```

**Comment by raoulvdberge** [comment]
This will be fixed by #3326

**Comment by raoulvdberge** [comment]
Hello. This bug has been fixed in v1.12.4 of Refined Storage. If this issue persists anyway, please feel free to create a new bug.

**Comment by raoulvdberge** [comment]
This bug has also been fixed in **v1.11.7** of Refined Storage for **Minecraft 1.19.2**. If this issue persists anyway, please feel free to create a new bug.

**Comment by raoulvdberge** [comment]
Hello, this bug is now also fixed for Minecraft 1.18.2 in v1.10.6.
